packages/gatsby-plugin-image/src/image-utils.ts

```diff
@@ -481,146 +481,146 @@ export function fixedImageSizes({
-    width = (height as number) * aspectRatio
-  }
-
-  const originalWidth = width
-  const isTopSizeOverriden =
-    imgDimensions.width < width || imgDimensions.height < (height as number)
-  if (isTopSizeOverriden) {
-    width = imgDimensions.width
-    height = imgDimensions.height
-  }
-
-  width = Math.round(width)
-
-  if (breakpoints?.length > 0) {
-    sizes = breakpoints.filter(size => size <= imgDimensions.width)
-
-    // If a larger breakpoint has been filtered-out, add the actual image width instead
-    if (
-      sizes.length < breakpoints.length &&
-      !sizes.includes(imgDimensions.width)
-    ) {
-      sizes.push(imgDimensions.width)
-    }
-  } else {
-    sizes = densities.map(density => Math.round(density * (width as number)))
-    sizes = sizes.filter(size => size <= imgDimensions.width)
-  }
-
-  // ensure that the size passed in is included in the final output
-  if (layout === `constrained` && !sizes.includes(width)) {
-    sizes.push(width)
-  }
-  sizes = sizes.sort(sortNumeric)
-  return {
-    sizes,
-    aspectRatio,
-    presentationWidth: originalWidth,
-    presentationHeight: Math.round(originalWidth / aspectRatio),
-    unscaledWidth: width,
-  }
-}
-
-export function getDimensionsAndAspectRatio(
-  dimensions,
-  options
-): { width: number; height: number; aspectRatio: number } {
-  // Calculate the eventual width/height of the image.
-  const imageAspectRatio = dimensions.width / dimensions.height
-
-  let width = options.width
-  let height = options.height
-
-  switch (options.fit) {
-    case `fill`: {
-      width = options.width ? options.width : dimensions.width
-      height = options.height ? options.height : dimensions.height
-      break
-    }
-    case `inside`: {
-      const widthOption = options.width
-        ? options.width
-        : Number.MAX_SAFE_INTEGER
-      const heightOption = options.height
-        ? options.height
-        : Number.MAX_SAFE_INTEGER
-
-      width = Math.min(widthOption, Math.round(heightOption * imageAspectRatio))
-      height = Math.min(
-        heightOption,
-        Math.round(widthOption / imageAspectRatio)
-      )
-      break
-    }
-    case `outside`: {
-      const widthOption = options.width ? options.width : 0
-      const heightOption = options.height ? options.height : 0
-
-      width = Math.max(widthOption, Math.round(heightOption * imageAspectRatio))
-      height = Math.max(
-        heightOption,
-        Math.round(widthOption / imageAspectRatio)
-      )
-      break
-    }
-
-    default: {
-      if (options.width && !options.height) {
-        width = options.width
-        height = Math.round(options.width / imageAspectRatio)
-      }
-
-      if (options.height && !options.width) {
-        width = Math.round(options.height * imageAspectRatio)
-        height = options.height
-      }
-    }
-  }
-
-  return {
-    width,
-    height,
-    aspectRatio: width / height,
-  }
-}
+// export function responsiveImageSizes({
```

Comment:
What's wrong with these code ?  I think it's unrelative change ([CI failed](https://app.circleci.com/pipelines/github/gatsbyjs/gatsby/78209/workflows/d80b4a00-0dd1-47c7-9b36-3b537705611c/jobs/923412)).

Reply:
Thanks! doing to many things at the same time 😅